pkg/playground/react-patterns.js

```diff
@@ -40,33 +40,41 @@ import { showMultiTypeaheadDemo } from "./react-demo-multi-typeahead.jsx";
 
 let lastAction = "";
 
-const onDialogStandardClicked = function(mode) {
+const onDialogStandardClicked = function(mode, progress_cb) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-22186-804e4a75-20250708-145413-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)